src/test/java/org/folio/fqm/ModFqmManagerApplicationTest.java

```diff
@@ -2,8 +2,7 @@
 
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
-import javax.validation.Valid;
-
+import jakarta.validation.Valid;
```

Comment:
called it, lol (a pointless javax -> jakarta)

Reply:
I immediately thought of your comment when this one came up 😄